dotCMS/src/main/java/com/dotmarketing/util/Logger.java

```diff
@@ -87,6 +87,12 @@ public static void debug(final Object ob, final Supplier<String> message) {
 		}
 	}
 
+	public static void debug(final Object ob, final Throwable throwable, final Supplier<String> message) {
+		if (isDebugEnabled(ob.getClass())) {
+			debug(ob.getClass(), message.get(), throwable);
```

Comment:
Wouldn't we want this to be logged like an error?
IDK how anti-pattern this would be: evaluate with `isDebugEnabled(...)` and then log with `error()` instead of `debug()`.